brd-android/kyc/src/main/java/com/fabriik/kyc/ui/features/accountverification/AccountVerificationViewModel.kt

```diff
@@ -77,7 +91,8 @@ class AccountVerificationViewModel(
                         KycStatus.KYC1,
                         KycStatus.KYC2_EXPIRED,
                         KycStatus.KYC2_DECLINED,
-                        KycStatus.KYC2_RESUBMISSION_REQUESTED ->
+                        KycStatus.KYC2_RESUBMISSION_REQUESTED,
+                        null ->
```

Comment:
Are we sure that we want to navigate to KYC2, if profile is null?

Reply:
Same as KYC1 comment.